e2e_tests_api/features/support/auth_base_url.rb

```diff
@@ -0,0 +1,25 @@
+module BaseAuth
+    include HTTParty
+    class << self
+
+        def access_token(scope)
+            @token = HTTParty.post('https://api-staging.infinitepay.io/v2/oauth/token' ,:body => {
+                "grant_type": "client_credentials", 
+                "client_id": "7851d7ef6cbf0b496cf84235492052c1",
+                "client_secret": "8d188f11489e80db162cd1f2b3a38b5b3ead315bc226b63e7d10a977b96a3e84",
```

Comment:
Can we avoid sharing `client_id` and `client_secret`? It is possible to add to github secrets?